### PR TITLE
ci: add SBOM generation

### PR DESCRIPTION
**Pull request by alegrey91**
## PR Type:
Enhancement

___
## PR Description:
This PR introduces the generation of Software Bill of Materials (SBOM) as part of the continuous integration (CI) process. The change is reflected in the 'pr-merged.yaml' workflow file, where the workflow now uses a new branch 'feat/add-sbom-creation' that presumably contains the necessary steps for SBOM creation.

___
## PR Main Files Walkthrough:
<details> <summary>files:</summary>

`.github/workflows/pr-merged.yaml`: The workflow file has been updated to use a new branch 'feat/add-sbom-creation' for the 'pr-merged' job. This suggests that the SBOM creation steps are included in this new branch.
</details>

___
## Overview
Closes: #165

### CODE REVIEW

File: .github/workflows/pr-merged.yaml [hunk 1]
```diff
@@ -9,15 +9,16 @@ on:
...
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   pr-merged:
-    if: ${{ github.event.pull_request.merged == true }} ## Skip if not merged
-    uses: kubescape/workflows/.github/workflows/incluster-comp-pr-merged.yaml@main
+    #if: ${{ github.event.pull_request.merged == true }} ## Skip if not merged
```
Comment (dwertent) [inline .github/workflows/pr-merged.yaml:20]: @alegrey91  Why did you remove this statement?

Comment (alegrey91) [reply .github/workflows/pr-merged.yaml:20]: Testing purposes. I'll remove this once the workflow works.
